tests/perf-test/ReportActionsList.perf-test.js

```diff
@@ -98,6 +96,7 @@ function ReportActionsListWrapper() {
     );
 }
 
+const runs = 20;
```

Comment:
nit: missing a new line right before the test case

Reply:
fixed ;-)